### ISSUE TITLE
Free Worlds Bounty Hunting: "Silverhawk" has old Hai outfits

### ISSUE THREAD

**Issue by Bladewood** [endless-sky/endless-sky#1426]
Human anti-missile turret, S-970 regenerator, fusion reactor, and A370 atomic thrusters and A525 atomic steering.
Surely, I'm not the only one who sees a problem here?

On an unrelated note, the Silverhawk's pulse cannons' descriptions talk about the Hai... when I haven't yet entered Hai space this run...

**Comment by Amazinite** [comment]
Well the mission and variant name describes it as having "alien weapons," not entirely alien outfits.

**Comment by KiLEdEnNis** [comment]
I never even saw the mission and I played 500 hours :/

**Comment by Bladewood** [comment]
I found it about a month ago while digging through the files.

**Comment by LocalGod79** [comment]
wow...totally forgot about that one while preparing the Hai outfits.

we definitely need to update the shield regenerator, but i think the rest of it can go either way...i'll wait for some community input.

as for the outfit description (Pulse Cannon), this spoiler has always been present - and offers an early game player a teaser of aliens that are on the other side of the galaxy (and perhaps inspires exploration if it wasn't already present).  that it's offered in the south means that the player would typically have to do a lot of exploring to actually find the Hai (if they didn't already know where they were)...

**Comment by Sinsling** [comment]
Aren't the s-series generators suppose to become available at some point for syndicates or something as less efficient human versions of Hai generators?

**Comment by Bladewood** [comment]
In the Checkmate route, yes.
If you choose to play the Reconcilation route, you can opt to instead hunt Marauder ships.

**Comment by LocalGod79** [comment]
the S-970 is only on the Marauder Firebirds though, so you may have to grind a bit to stock up on those...

with that said, the mission in the o.p.'s question is part of `fw intro.txt`, at the opposite end of the checkmate arc (where the Syndicate regenerators are unlocked) - so _they_ are the most incongruous outfit on the Silverhawk, and definitely need to be changed.

the question that i'm coming up with while preparing the outfits:
```
		"Ion Cannon" 2
		"Pulse Cannon" 2
		"Bullfrog Anti-Missile" 2
		"Boulder Reactor"
		"Hai Fissure Batteries"
		"Hai Diamond Regenerator"
		"Liquid Nitrogen Cooler"
		`"Bondir" Atomic Thruster`
		`"Bondir" Atomic Steering`
		Hyperdrive
```
or
```
		"Ion Cannon" 2
		"Pulse Cannon" 2
		"Pulse Turret"
		"Chameleon Anti-Missile"
		"Boulder Reactor"
		"Hai Fissure Batteries"
		"Hai Corundum Regenerator"
		"Liquid Nitrogen Cooler"
		`"Bondir" Atomic Thruster`
		`"Bondir" Atomic Steering`
		Hyperdrive
```

**Comment by KevinKorzekwa** [comment]
I like the Chameleon, Pulse Turret, and Corundum.  I like the flexibility of having damage turrets paired with cannons.  And the shield regen from the Corundum generator is not insignificant.

**Comment by jafdy** [comment]
I would go for the pair of Bullfrogs and a Diamond. My shield would regenerate faster. (and I like symmetrical ships)

**Comment by KiLEdEnNis** [comment]
diamond basically triples recharge, although it'd be a bit less strong in damage, the ion cannons make up for that though

**Comment by Alkallid** [comment]
I thought I told you about this, and Wrzl too? I remember this being done already.

**Comment by jafdy** [comment]
While you may have told them it hasn't been done.

**Comment by endless-sky** [comment]
Addressed in commit 3e592ca.

**Comment by endless-sky** [comment]
Closing this because 0.9.5 is released now.
